README.md

```diff
@@ -2,29 +2,77 @@
 
 This is a scraper to collect and process public case records from the Tyler Technologies Odyssey court records system. If you are a dev or want to file an Issue, please read [CONTRIBUTING](CONTRIBUTING.md).
 
-## Install
+## Local setup
+
+### Install toolchain
 
 1. Clone this repo and navigate to it.
    - `git clone https://github.com/open-austin/indigent-defense-stats`
    - `cd indigent-defense-stats`
-1. With pip, install the requirements.
-   - `pip install -r requirements.txt`
-1. Set any parameters.
+2. Install Pyenv if not already installed ([linux, mac](https://github.com/pyenv/pyenv), or [windows](https://github.com/pyenv-win/pyenv-win))
+3. Run `pyenv install` to get the right Python version
+
+### Setup `venv`
+
+First, you'll need to create a virtual environment, this differs depending on your OS.
+
+On linux/mac
+
+```bash
+python -m venv .venv --prompt ids # (you can substitute `ids` for any name you want)
+```
+
+On Windows
+
+```powershell
+c:\>Python35\python -m venv c:\path\to\repo\ids # (you can substitute `ids` for any name you want)
+```
+
+Next, you'll need to "activate" the venv. You'll need to run this command every time you work in the codebase and tell your IDE which Python environment to use. It will likely default to wherever `python` resolves to in your system path. The specific command you run will depend on both your OS and shell.
+
+On linux/mac
+
+| platform | shell      | Command to activate virtual environment |
+| :------- | :--------- | :-------------------------------------- |
+| POSIX    | bash/zsh   | $ source <venv>/bin/activate            |
+|          | fish       | $ source <venv>/bin/activate.fish       |
+|          | csh/tcsh   | $ source <venv>/bin/activate.csh        |
+|          | PowerShell | $ <venv>/bin/Activate.ps1               |
+| Windows  | cmd.exe    | C:\> <venv>\Scripts\activate.bat        |
+|          | PowerShell | PS C:\> <venv>\Scripts\Activate.ps1     |
+
+Source: https://docs.python.org/3/library/venv.html#how-venvs-work
+
+Note: Again, you'll need to activate venv _every time you want to work in the codebase_.
+
+### Install python dependencies
+
+Using `pip`, install the project dependencies.
+
+```shell
+pip install -r requirements.txt
+```
+
+### Running CLI
+
+@TODO - this section needs to be updated.
```

Comment:
@nicolassaw The command mentioned here might still be working, but I suspect it might break when we refactor the module structure.